imap_processing/swapi/l2/swapi_l2.py

```diff
@@ -0,0 +1,115 @@
+"""SWAPI L2 processing module."""
+
+import logging
+
+import xarray as xr
+
+from imap_processing.cdf.imap_cdf_manager import ImapCdfAttributes
+from imap_processing.swapi.swapi_utils import SWAPIAPID
+
+logger = logging.getLogger(__name__)
+
+
+def process_science(l1_dataset: xr.Dataset, data_version: str) -> xr.Dataset:
+    """
+    Produce science data to L2.
+
+    To process science data to L2, we need to:
+    - convert counts to rates. This is done by dividing the counts by the
+        t_bin time. t_bin is the exposure time per energy bin which is
+        obtained by dividing the time for one complete sweep
+        (12 s, coarse + fine sweep) by the total energy steps (72),
+        i.e., t_bin = 12/72 = 0.167 s. This will be constant.
```

Comment:
Make this a global constant outside the function? `TIME_PER_BIN = 12 / 72`

Note that 12/72 is not equal to 0.167, it is 0.166666 repeating. Do you want to store 0.167 or 12/72?

Reply:
Their algorithm document says to use `0.167`. That's why I used that. This comment was to give background understanding of where that value came from. I can ask SWAPI team this week.